contrib/julia-config.jl

```diff
@@ -23,6 +23,8 @@ function libDir()
     end
 end
 
+private_libDir() = joinpath(libDir(), "julia")
```

Comment:
might have to worry about running this file while also having started julia with `-J` ?